drivers/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,12 @@
+# BH1750 ambient light sensor configuration options
+
+# Copyright (c) 2022 morsisko
```

Comment:
```suggestion
# Copyright (c) 2022 Michal Morsisko
```

Afaik we require real names

Reply:
I saw this as well but I believe that is only for Signed-off-by: lines. I don't know that we have a formal policy for copyright lines.